### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by Shrinks99**
### Notable Changes to Readme.md
- Removed broken Coverity scan button
- Removed Open Hub button (the amount of developers it features is incorrect, this is listed right in the sidebar on GitHub, and it looks ugly!
- All buttons are now using SVGs and look nice and sharp!
- Added Discord link
- Reworked headings to use actual H1s in Markdown
- Lots of minor rewording, made some sections more concise, fixed grammar
- Standardized bullet points as dashes
- Removed "Planned Features" section.  Any actually planned features should be done with a roadmap / GitHub task board so they can be linked to issues and closed accordingly, also I'm pretty sure none of these are actively planned right now.
- Lots of spacing standardization, makes this file a little difficult to review.  Sorry! :(
- Updates tools section and links to the tools directory which has since been overhauled with its own readme
- Removes download links to sourceforge, links to releases page instead.

Readme.md can be previewed [here](https://github.com/Shrinks99/Natron/tree/readme-update)

<del>### Notable Changes to INSTALL_OSX.md</del>
<del>- Renamed file to `INSTALL_MACOS.md`</del>
  <del>- Will this have problems? I made sure to relink on the readme, anywhere else I'm missing?</del>
<del>- Added syntax highlighting and fixed indenting for every command!  GitHub's syntax highlighting is somewhat lackluster but hey, it's in there!</del>
<del>- Fixed capitalization of "Homebrew", this is now standardized!</del>
<del>- Didn't actually fix the build process :(</del>

<del>INSTALL_MACOS.md can be previewed [here](https://github.com/Shrinks99/Natron/blob/readme-update/INSTALL_MACOS.md)</del>

### CODE REVIEW

File: INSTALL_MACOS.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 
-OS X 10.6 (a.k.a. Snow Leopard) and newer are supported when building with MacPorts, and Homebrew can be used to compile it on the latest OS X.
+macOS 10.6 (a.k.a. Snow Leopard) and newer are supported when building with MacPorts, and Homebrew can be used to compile it on the latest version of macOS.
 
 ## Official Natron binaries
 
 The official Natron and plugins binaries are built using the section about MacPorts in these instructions to prepare the system, and the `launchBuildMain.sh` build script found in the `tools/jenkins` directory. The script takes care of everything, from checking out sources, to compiling and packaging.
 
-These binaries are built on an OS X 10.9 (Mavericks) virtual machine with [Xcode 6.2](https://developer.apple.com/devcenter/download.action?path=/Developer_Tools/Xcode_6.2/Xcode_6.2.dmg). Note that [Mavericks can not be downloaded anymore from the 10.14 (High Sierra) App Store](https://www.macworld.co.uk/how-to/mac-software/download-old-os-x-3629363/), so you will need to use a Mac with an older system (up to 10.13), or look for [alternatives](https://applehint.com/t/download-all-macos-x-10-4-10-14-original/376).
```
Comment (devernay) [inline INSTALL_MACOS.md:12]: Mavericks is actually OS X Mavericks, not macOS https://en.wikipedia.org/wiki/OS_X_Mavericks

File: INSTALL_MACOS.md [hunk 2]
```diff
@@ -35,48 +34,60 @@ Homebrew is easier to set up than MacPorts, but cannot build universal binaries.
...
+```
 
 It is also recommended to add the  following line to `/opt/local/etc/macports/variants.conf`:
 
-    -x11 +no_x11 +bash_completion +no_gnome +quartz +natron
+```
+-x11 +no_x11 +bash_completion +no_gnome +quartz +natron
+```
 
-#### Notes for OS X 10.6 Snow Leopard
```
Comment (devernay) [inline INSTALL_MACOS.md:53]: This is actually Mac OS X Snow Leopard https://en.wikipedia.org/wiki/Mac_OS_X_Snow_Leopard 
Don't blame me, blame Apple for the name changes!

File: INSTALL_MACOS.md [hunk 3]
```diff
@@ -96,159 +107,189 @@ EOF
...
-    sudo ln -s ~/Library/Python/2.7/lib/python/site-packages/homebrew.pth /Library/Python/2.7/lib/python/site-packages/homebrew.pth
+```shell
+mkdir -p ~/Library/Python/2.7/lib/python/site-packages
+echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> ~/Library/Python/2.7/lib/python/site-packages/homebrew.pth
+sudo ln -s ~/Library/Python/2.7/lib/python/site-packages/homebrew.pth /Library/Python/2.7/lib/python/site-packages/homebrew.pth
+```
 
  To install the [openfx-io](https://github.com/NatronGitHub/openfx-io) and [openfx-misc](https://github.com/NatronGitHub/openfx-misc) sets of plugin, you also need the following:
 
-    brew install ilmbase openexr freetype fontconfig ffmpeg opencolorio openimageio
```
Comment (devernay) [inline INSTALL_MACOS.md:169]: Natron won't compile with the most recent version of seexpr, why was this part removed?

File: INSTALL_MACOS.md [hunk 4]
```diff
@@ -96,159 +107,189 @@ EOF
...
+```
 
 Then, configure using:
 
-    ./configure -prefix /opt/qt4 -pch -system-zlib -qt-libtiff -qt-libpng -qt-libjpeg -confirm-license -opensource -nomake demos -nomake examples -nomake docs -cocoa -fast -release
+```shell
+./configure -prefix /opt/qt4 -pch -system-zlib -qt-libtiff -qt-libpng -qt-libjpeg -confirm-license -opensource -nomake demos -nomake examples -nomake docs -cocoa -fast -release
+```
 
-* On OS X >= 10.9 add `-platform unsupported/macx-clang-libc++`
```
Comment (devernay) [inline INSTALL_MACOS.md:216]: this is OS X

**Review by devernay — CHANGES_REQUESTED** [review]
Thank you for the README.md changes.

However, the INSTALL_OSX changes are not very useful, and I don't think we can accept them, since it cannot be used to build NATRON.
- why remove the part about seexpr version?
- the Apple desktop OS was named Mac OS, then Mac OS X, then OS X, then macOS, so that each version has its own name. It should be keps that way
- About "Colour" vs "Color": the Natron code and doc are (mostly) written in US english, probably with some French here and there, so it should really be "Color". The places where it's "Colour" are because Foundry (which used to be called "The Foundry") is a UK company.
- most of the changes consist in markdown reformating

Could you please resubmit with just the README.md changes? (in general, try to split PRs, that's easier for review)

File: INSTALL_MACOS.md (hunk 3 above)
Comment (Shrinks99) [reply INSTALL_MACOS.md:169]: This was an error, will fix.

File: INSTALL_MACOS.md (hunk 2 above)
Comment (Shrinks99) [reply INSTALL_MACOS.md:53]: Oh... I do ;P

**Comment by Shrinks99** [comment]
Reverted INSTALL_OSX, should be good to go!

**Review by devernay — APPROVED** [review]
